_commands/tools/find.md

```diff
@@ -3,22 +3,195 @@
 
 find
 ---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`find` is used to search one or more directory trees of a file system, to locate files based on user-defined specifications and to apply a user-defined action on each matched target. 
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ find
```

Comment:
I mentioned the equivalence to `find .` in the first example `find` in **Useful Options / Examples**. Do you want me to also include that in the short example?

Reply:
The point I was trying to make was that `find` without any parameters is not really a valid command. If someone was unfamiliar with this command, they might try typing just `find` into their own terminal and get entirely different output, so yes, the short example should be something like `find .`

Reply:
Thank you for pointing that out. I pushed another commit so that every example usage of `find` (including short example) has a path as parameter, which is consistent with what is given by `man find`.